cmd/podman/diff/diff.go

```diff
@@ -73,6 +73,9 @@ func ValidateContainerDiffArgs(cmd *cobra.Command, args []string) error {
 		return errors.New("--latest and containers cannot be used together")
 	}
 	if len(args) == 0 && !given {
+		if !registry.IsRemote() {
+			return fmt.Errorf("%q requires a name, id", cmd.CommandPath())
```

Comment:
requires a name or id.

Reply:
@rhatdan 
Thank you for pointing that out.
I have corrected the grammatical incorrectness of the error message.